### PR TITLE
Option to use system emoji

### PR DESCRIPTION
**Pull request by mcginty**

### CODE REVIEW

File: src/org/thoughtcrime/securesms/components/emoji/EmojiEditText.java [hunk 1]
```diff
@@ -24,7 +25,9 @@ public EmojiEditText(Context context, AttributeSet attrs) {
 
   public EmojiEditText(Context context, AttributeSet attrs, int defStyleAttr) {
     super(context, attrs, defStyleAttr);
-    setFilters(new InputFilter[]{ new EmojiFilter(this) });
+    if (!TextSecurePreferences.isSystemEmojiPreferred(getContext())) {
```
Comment (moxie0) [inline src/org/thoughtcrime/securesms/components/emoji/EmojiEditText.java]: You're not nervous about i/o implications here?

Comment (mcginty) [inline src/org/thoughtcrime/securesms/components/emoji/EmojiEditText.java]: I am, but it should be cached after the first read, meaning only one I/O hit per app run. I kept it simple so didn't make an async task, but still possible if need be.

**Comment by mcginty** [comment]
looks like this is in 2.27.0

**Comment by turtlekiosk** [comment]
omg it's really happening!!!

**Pull request closed without merging**